client/src/redux/actions/tests/scansActions.test.js

```diff
@@ -53,14 +53,17 @@ describe('ScansActions', function() {
       });
     });
```

Comment:
Note: Next round we could look at moving the ```moxios.wait...``` up to the ```beforeEach```, unless of course it needs to be customized